pkg/odo/cli/component/delete.go

```diff
@@ -48,12 +48,13 @@ type DeleteOptions struct {
 	// devfile path
 	devfilePath     string
 	namespace       string
+	show            bool
 	EnvSpecificInfo *envinfo.EnvSpecificInfo
 }
 
 // NewDeleteOptions returns new instance of DeleteOptions
 func NewDeleteOptions() *DeleteOptions {
-	return &DeleteOptions{false, false, false, false, "", false, &ComponentOptions{}, "", "", nil}
+	return &DeleteOptions{false, false, false, false, "", false, &ComponentOptions{}, "", "", false, nil}
```

Comment:
Since you're adding another parameter to the struct, could we reformat this to make it a little more readable and to see which parameters are getting set?
```
return &DeleteOptions{
    componentForceDeleteFlag: false, 
    componentDeleteAllFlag: false,
    componentDeleteWaitFlag: false,
    componentDeleteS2iFlag: false, 
    componentContext: "", 
    isCmpExists: false, 
    ...
}

Reply:
good point, updated

Reply:
@johnmcollier pls help review the PR, thx

Reply:
👍